### PR TITLE
Changed comment_data_validator

### PR DESCRIPTION
**Pull request by skramar**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [x] @dmytrorevak
- [ ] @IvanAndrukh
- [x] @JjuliaP
- [x] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by redmyt — APPROVED** [review]

File: eventually/comment/views.py [hunk 1]
```diff
@@ -88,8 +88,8 @@ def post(self, request, team_id=None, event_id=None, task_id=None, vote_id=None)
             return HttpResponse(status=404)
 
         data = request.body
-        if not comment_data_validator(data):
-            return HttpResponse(status=400)
+        if not comment_data_validator(data, required_keys=['text']):
+            return HttpResponse('sadasd', status=400)
```
Comment (redmyt) [inline eventually/comment/views.py]: what is `'sadasd'`? =)

File: eventually/utils/validators.py [hunk 2]
```diff
@@ -295,24 +295,46 @@ def login_validate(data):
...
-    required_keys = ["text"]
-    if not required_keys_validator(data, required_keys):
-        return
+    errors = []
+    if not required_keys_validator(data=data, keys_required=required_keys, strict=False):
+        errors.append('required keys error')
 
-    if not string_validator(data.get("text")):
-        return
+    comment_model_fields = ['text',
```
Comment (redmyt) [inline eventually/utils/validators.py:316]: are there tests for this function?

**Review by redmyt — CHANGES_REQUESTED** [review]

File: eventually/comment/views.py (hunk 1 above)
Comment (skramar) [reply eventually/comment/views.py]: done

File: eventually/utils/validators.py (hunk 2 above)
Comment (skramar) [reply eventually/utils/validators.py:316]: nope

**Review by JjuliaP — APPROVED** [review]

**Review by makarman — APPROVED** [review]

**Review by redmyt — APPROVED** [review]

**Pull request merged**